### ISSUE TITLE
MissingIDFieldWarning: Code cell is missing an id field

### ISSUE THREAD

**Issue by awaelchli** [Lightning-AI/tutorials#198]
## 🐛 Bug

We're seeing an error from the docs build when notebooks get generated in the Lightning CI.

### To Reproduce

The error can be observed on the latest PL docs build:
```
reading sources... [ 85%] notebooks/course_UvA-DL/01-introduction-to-pytorch
/opt/hostedtoolcache/Python/3.9.14/x64/lib/python3.9/site-packages/nbformat/__init__.py:92: MissingIDFieldWarning: Code cell is missing an id field, this will become a hard error in future nbformat versions. You may want to use `normalize()` on your notebooks before validations (available since nbformat 5.1.4). Previous versions of nbformat are fixing this issue transparently, and will stop doing so in the future.
  validate(nb)
```

Example CI job with errors:
https://github.com/Lightning-AI/lightning/actions/runs/3129233095/jobs/5078064605

### Expected behavior

Passes

### Additional context

Environment in CI:

```
Package                       Version           Editable project location
----------------------------- ----------------- -------------------------------------
absl-py                       1.2.0
aiohttp                       3.8.3
aiosignal                     1.2.0
alabaster                     0.7.12
argon2-cffi                   21.3.0
argon2-cffi-bindings          21.2.0
asttokens                     2.0.8
async-timeout                 4.0.2
attrs                         22.1.0
Babel                         2.10.3
backcall                      0.2.0
beautifulsoup4                4.11.1
bleach                        5.0.1
bracex                        2.3.post1
cachetools                    5.2.0
certifi                       2022.9.24
cffi                          1.15.1
charset-normalizer            2.1.1
debugpy                       1.6.3
decorator                     5.1.1
defusedxml                    0.7.1
docutils                      0.17.1
entrypoints                   0.4
executing                     1.1.0
fastjsonschema                2.16.2
fire                          0.4.0
psutil                        5.9.2
pt-lightning-sphinx-theme     0.0.30
ptyprocess                    0.7.0
pure-eval                     0.2.2
pyasn1                        0.4.8
pyasn1-modules                0.2.8
pycparser                     2.21
Pygments                      2.13.0
pyparsing                     3.0.9
pyrsistent                    0.18.1
python-dateutil               2.8.2
pytorch-lightning             1.8.0.dev0        /home/runner/work/lightning/lightning
pytz                          2022.2.1
PyYAML                        6.0
pyzmq                         24.0.1
requests                      2.28.1
requests-oauthlib             1.3.1
rsa                           4.9
Send2Trash                    1.8.0
setuptools                    58.1.0
six                           1.16.0
snowballstemmer               2.2.0
soupsieve                     2.3.2.post1
Sphinx                        4.5.0
sphinx-autodoc-typehints      1.14.1
sphinx-copybutton             0.5.0
sphinx-multiproject           1.0.0rc1
sphinx-paramlinks             0.5.4
sphinx-togglebutton           0.3.2
sphinxcontrib-applehelp       1.0.2
sphinxcontrib-devhelp         1.0.2
sphinxcontrib-fulltoc         1.2.0
sphinxcontrib-htmlhelp        2.0.0
sphinxcontrib-jsmath          1.0.1
sphinxcontrib-mockautodoc     0.0.1.dev20130[518](https://github.com/Lightning-AI/lightning/actions/runs/3129233095/jobs/5078064605#step:5:519)
sphinxcontrib-qthelp          1.0.3
sphinxcontrib-serializinghtml 1.1.5
stack-data                    0.5.1
tensorboard                   2.10.1
tensorboard-data-server       0.6.1
tensorboard-plugin-wit        1.8.1
termcolor                     2.0.1
terminado                     0.15.0
tinycss2                      1.1.1
torch                         1.12.1+cpu
torchmetrics                  0.9.2
tornado                       6.2
tqdm                          4.64.1
traitlets                     5.4.0
typing_extensions             4.3.0
urllib3                       1.26.12
wcmatch                       8.4.1
wcwidth                       0.2.5
webencodings                  0.5.1
Werkzeug                      2.2.2
wheel                         0.37.1
widgetsnbextension            4.0.3
yarl                          1.8.1
zipp                          3.8.1
```

**Comment by Borda** [comment]
it shall be addressed in #212

**Comment by awaelchli** [comment]
This still seems to be an issue. Right now, I believe it is blocking the read the docs build, and preventing latest changes to be published in docs: https://readthedocs.org/projects/pytorch-lightning/builds/19129763/

**Comment by Borda** [comment]
> it shall be addressed in #212

ok, it seems to be still there but notebooks pass...
